src/components/FeeExplainer/FeeExplainer.tsx

```diff
@@ -258,42 +257,30 @@ export type FeeSlidersProps = {
 }
 
 type FeeOutputProps = {
-  tradeSize: number
+  tradeSizeUSD: number
```

Comment:
nitpick but we seems to use `xxxUsd` rather than `xxxUSD` for naming everywhere else, should we do that too here?